src/editor.cpp

```diff
@@ -2056,22 +2060,23 @@ void GetEditorControls()
 {
     if(HasCursor)
     {
-        if((getKeyState(vbKeyUp) == KEY_PRESSED))
-            EditorControls.Up = true;
-        else
-            EditorControls.Up = false;
-        if((getKeyState(vbKeyDown) == KEY_PRESSED))
-            EditorControls.Down = true;
-        else
-            EditorControls.Down = false;
-        if((getKeyState(vbKeyLeft) == KEY_PRESSED))
-            EditorControls.Left = true;
-        else
-            EditorControls.Left = false;
-        if((getKeyState(vbKeyRight) == KEY_PRESSED))
-            EditorControls.Right = true;
-        else
-            EditorControls.Right = false;
+        // TODO: move elsewhere
+        // if((getKeyState(vbKeyUp) == KEY_PRESSED))
+        //     EditorControls.Up = true;
+        // else
+        //     EditorControls.Up = false;
+        // if((getKeyState(vbKeyDown) == KEY_PRESSED))
+        //     EditorControls.Down = true;
+        // else
+        //     EditorControls.Down = false;
+        // if((getKeyState(vbKeyLeft) == KEY_PRESSED))
+        //     EditorControls.Left = true;
+        // else
+        //     EditorControls.Left = false;
+        // if((getKeyState(vbKeyRight) == KEY_PRESSED))
+        //     EditorControls.Right = true;
+        // else
+        //     EditorControls.Right = false;
```

Comment:
Are you sure you didn't break anything?

Reply:
Definitely broke the editor controls for now. Just want to move all of the EditorControls logic into a single place closer to `joystick.cpp` and not in `editor.cpp`. Which is why I wrote TODO.